meeting_rooms/tests.py

```diff
@@ -80,31 +98,51 @@ def test_04_post_payload_negative_capacity(self):
 
 
 class TestRequestGet(APITestCase):
+    def setUp(self):
+        self.meeting_room = MeetingRoom.objects.create(
+            room_number='1', capacity=30
+        )
+
     def test_05_request_get_all_meeting_rooms(self):
         # Test valid GET request -- all meeting rooms
         response = self.client.get(ENDPOINT)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            response.status_code, status.HTTP_200_OK, response.json()
+        )
 
     def test_06_request_get_single_meeting_room(self):
         # Test valid GET request -- single meeting room
-        response = self.client.get(ENDPOINT+'1')
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response = self.client.get(ENDPOINT + f'{self.meeting_room.id}/')
+        self.assertEqual(
+            response.status_code, status.HTTP_200_OK, response.json()
+        )
 
     def test_07_request_get_single_meeting_room_invalid_id(self):
         # Test valid GET request -- object does not exist
-        invalid_id = str(MeetingRoom.objects.latest('id').id + 1)
-        response = self.client.get(ENDPOINT+invalid_id)
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        id = str(MeetingRoom.objects.latest('id').id + 1)
```

Comment:
Maybe for tests it's not so relevant, but still worth mentioning (won't call the database for a second time):
```suggestion
        id = str(self.meeting_room.id + 1)
```